src/Resources/config/makers.xml

```diff
@@ -27,6 +27,7 @@
             <service id="maker.maker.make_crud" class="Symfony\Bundle\MakerBundle\Maker\MakeCrud">
                 <argument type="service" id="maker.doctrine_helper" />
                 <argument type="service" id="maker.renderer.form_type_renderer" />
+                <argument key="$attributeMappingSupport">%maker.compatible_check.symfony.supports_attributes%</argument>
```

Comment:
The name loosk wrong here:

```suggestion
                <argument key="$attributeMappingSupport">%maker.compatible_check.doctrine.supports_attributes%</argument>
```

Moreover, I think this argument should be added to the definition of `DoctrineHelper` and not `MakeCrud`?

Reply:
Somehow its find its way in there... I removed it and the `DoctrineHelper` gets the parameter now.